src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -2035,13 +2038,6 @@ int wm_vuldet_report_agent_vulnerabilities(sqlite3 *db, agent_software *agent, w
         goto end;
     }
 
-    int sock = wm_vuldet_get_wdb_socket();
```

Comment:
The removal of this set of lines will cause the Vulnerability Detector unit tests to fail. We should decide whether to fix them here or in a separate PR.